style/reading/links-to-physics.less

```diff
@@ -19,15 +20,11 @@
     text-transform: uppercase;
     letter-spacing: 2px;
     text-align: center;
-    background: @physics-background-secondary;
+    background: (@physics-background-secondary);
 
-    &:before {
-      content: "\2014  ";
-    }
+    &:before { content: "\2014  ";}
```

Comment:
Can you include a comment identifying which character this is supposed to be? Do the 2 spaces matter? and this seems to be identical to code in `work-in-physics.less`; should they be combined?